tests/rados/test_slow_op_requests.py

```diff
@@ -0,0 +1,85 @@
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados import utils
+from ceph.rados.core_workflows import RadosOrchestrator
+from tests.rados.rados_test_util import (
+    create_pools,
+    get_device_path,
+    get_slow_requests_log,
+    wait_for_device,
+    write_to_pools,
+)
+from tests.rados.stretch_cluster import wait_for_clean_pg_sets
+from utility.log import Log
+from utility.utils import method_should_succeed, should_not_be_empty
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Perform OSD remove operation to verify slow op requests
+    """
+    try:
+        log.info(run.__doc__)
+        config = kw["config"]
+        cephadm = CephAdmin(cluster=ceph_cluster, **config)
+        rados_obj = RadosOrchestrator(node=cephadm)
+        client_node = ceph_cluster.get_nodes(role="client")[0]
+        installer = ceph_cluster.get_nodes(role="installer")[0]
+
+        log.info("Running slow op requests tests")
+        pool = create_pools(config, rados_obj, client_node)
+        should_not_be_empty(pool, "Failed to retrieve pool details")
+        write_to_pools(config, rados_obj, client_node)
+        rados_obj.change_recover_threads(config=pool, action="set")
+        acting_pg_set = rados_obj.get_pg_acting_set(pool_name=pool["pool_name"])
+        log.info(f"Acting set {acting_pg_set}")
+        should_not_be_empty(acting_pg_set, "Failed to retrieve acting pg set")
+        osd_id = acting_pg_set[0]
+        host = rados_obj.fetch_host_node(daemon_type="osd", daemon_id=osd_id)
+        should_not_be_empty(host, "Failed to fetch host details")
+        dev_path = get_device_path(host, osd_id)
+        log.debug(
+            f"osd1 device path  : {dev_path}, osd_id : {osd_id}, host.hostname : {host.hostname}"
+        )
+
+        rados_obj.enable_file_logging()
```

Comment:
I think since we are using `journalctl` to fetch logs, this would not be required.

Reply:
When logging was not enabled, I was not able to fetch the slow op logs.